topics/admin/tutorials/ansible-galaxy/tutorial.md

```diff
@@ -441,6 +438,10 @@ For this tutorial, we will use the default "peer" authentication, so we need to
 >    ```
 >    {: data-commit="Add initial group variables file"}
 >
+>    > <comment-title>Regarding `expanduser`</comment-title>
+>    > The `expanduser` filter turns `~postgres` into the absolute path to the `postgres` user's home directory. However, you should be warned that the `expanduser` filter, like all Ansible filters, runs *locally*. This works in the training because Ansible is executed from the Galaxy server, but if you run Ansible over SSH, you would need to set the full path in `postgresql_backup_local_dir` directly.
```

Comment:
this has bitten me before. I don't know if I like it. Perhaps we should just explicitly set to the full path? I expect 1 out of 20 students remembers this comment when they follow this training and later experience it.

Reply:
Ok, I agree with that (we're already reaching the point of oversaturation of tips/comments in this tutorial anyway).

Kinda silly that Ansible still has no built in mechanism for this. You have to write a series of tasks that run the getent module and extract the attributes you want to facts. So here we can just hardcode the path.